Revised TechDoc Analysis Tools (#229)
cncf/techdocs: README.md

# CNCF TechDocs

This repository holds resources provided by the CNCF Technical Documentation team. The repo contains the following directories:

- `analysis` contains instructions, templates, and criteria for requesting and performing an analysis of an OSS project's website and technical documentation. Completed analyses are stored here as well.

> - `resources` contains information that OSS teams can use to set up a documentation project as suggested by the Tech Docs team.

Reviewer: We've traditionally used "TechDocs" 🤷‍♂️, so let's stick to that spelling everywhere, i.e. `s/Tech Docs/TechDocs/g` in all the updates, like this:

```suggestion
- `resources` contains information that OSS teams can use to set up a documentation project as suggested by the TechDocs team.
```

Author: I've always thought camel-case spelling was used mostly for brand names -- not something I'd apply to a department name or functional description. I have fairly conservative stylistic taste, though. If you're using it everywhere else then make it universal. Consistency is important as well.

### Meeting notes

We store ongoing meeting notes in a [Google doc](https://docs.google.com/document/d/1roexHTLCrErYjNT2NEoRsVnn_YNbQzZ1gyXNK8hXR4Q/). 

## Assistance program for technical documentation 

The TechDocs team can assist CNCF projects analyze and improve their documentation. For details, see the TechDocs [assistance-program][].

> [assistance-program]: ./techdoc-assistance.md

Reviewer 2: Where should this link now?

### Meeting notes

We store ongoing meeting notes in a [Google doc](https://docs.google.com/document/d/1roexHTLCrErYjNT2NEoRsVnn_YNbQzZ1gyXNK8hXR4Q/). 

> ## The Technical Documentation Assistance Program

Reviewer: ```suggestion
## Assistance program for technical documentation 
```

> The CNCF Tech Doc team has created a program to assist CNCF-affiliated projects in creating effective documentation. [`TechDoc-Assistance-Program.md`](./TechDoc-Assistance-Program.md) contains an outline of this program. 
> 
> Project maintainers are asked to understand that:
> - Project documentation resources are *always* underestimated.
> - Documentation is as much a real part of a project as code. 
> - Effective documentation requires a real commitment of resources. 
> - The TechDoc Assistance Program exists to help maintainers allocate these resources effectively, but cannot write documentation for you or 'fix' documentation problems. 
> 
> We're passionate about good documentation and we want to help your project succeed. Please work with us by taking this part of your project seriously.

Reviewer: I would suggest gentler language; see below.

As for the pitch, importance, and realities surrounding documentation, if we do mention anything, we should link out to external resource(s) IMHO.

```suggestion
The TechDocs team can assist CNCF projects analyze and improve their documentation. For details, see TechDocs [Assistance program](./TechDoc-Assistance-Program.md).
```

Author: I agree, my wording was a bit provocative.
